jerry-core/debugger/jerry-debugger.h

```diff
@@ -16,24 +16,72 @@
 #ifndef JERRY_DEBUGGER_H
 #define JERRY_DEBUGGER_H
 
-#define MAX_MESSAGE_SIZE 128
+#define MAX_BUFFER_SIZE 128
+
+char jerry_debugger_buffer[MAX_BUFFER_SIZE];
+
+/**
+ * Package header
+ */
+typdef struct
+{
+  jerry_debugger_header_type_t type;        /**< type of the message */
+  uint8_t size;                             /**< size of the message */
+} jerry_debugger_message_header_t;
 
 /**
-* Package header
-*/
+ * Types for the package
+ *
+ * It can help for the debugger who can decide what type
+ * of the data come from the JerryScript. If the buffer size it's not enough for
+ * the message the engine split up into many pieces. The last piece we know if the
```

Comment:
1. splits it up into ...
2. The header of the last piece has a type with an '_END' postfix.